src/Components/HourlyWeather/index.js

```diff
@@ -0,0 +1,53 @@
+import { useEffect, useState } from "react";
+import "./index.css";
+
+const HourlyWeather = ({ placename }) => {
+    const [hourlyData, setHourlyData] = useState([]);
+
+    useEffect(async () => { 
+        async function convertGeoLocation(placename) {
+            const response = await fetch(`https://api.opencagedata.com/geocode/v1/json?q=${placename}&key=${process.env.OPEN_CAGE_APIKEY}`).then(value => value.json());
+            const cord = response["results"][0]["geometry"];
```

Comment:
Make the useEffect function synchronous. This is a bad practice.

Make it something like this:

```
 async function apiCallsFn() {
    // Your Code

  }
  
  
  useEffect( () => {
    
    apiCallsFn();
  }, [placename]);
 ```